compiler/il/OMROpcodes.enum

```diff
@@ -8229,6 +8229,134 @@ OPCODE_MACRO(\
    /* .ifCompareOpCode      = */ TR::BadILOp, \
    /* .description          = */ \
 )
+OPCODE_MACRO(\
+   /* .operation            = */ bcompressbits, \
+   /* .name                 = */ "bcompressbits", \
+   /* .properties1          = */ 0, \
+   /* .properties2          = */ ILProp2::SupportedForPRE, \
+   /* .properties3          = */ 0, \
+   /* .properties4          = */ 0, \
+   /* .dataType             = */ TR::Int8, \
+   /* .typeProperties       = */ ILTypeProp::Size_1 | ILTypeProp::Integer, \
+   /* .childProperties      = */ TWO_SAME_CHILD(TR::Int8), \
+   /* .swapChildrenOpCode   = */ TR::BadILOp, \
+   /* .reverseBranchOpCode  = */ TR::BadILOp, \
+   /* .booleanCompareOpCode = */ TR::BadILOp, \
+   /* .ifCompareOpCode      = */ TR::BadILOp, \
+   /* .description          = compress bits: equivalent to calling Integer.compress(a, b) */
```

Comment:
It seems bit ambiguous what Integer.compress(a,b) is looking at the change just with OMR perspective.